app/views/simplified-account/settings/team-members/index.njk

```diff
@@ -0,0 +1,104 @@
+{% extends "../../../layout.njk" %}
+
+{% macro teamMemberEmail(email, isCurrent, isServiceAdmin) %}
+  <p class="govuk-body">{{email}}{% if isCurrent %} (you){% endif %}</p>
+{% endmacro %}
+
+{% block pageTitle %}
+  Settings - Team members
+{% endblock %}
+
+{% block sideNavigation %}
+  {% include "../_settings-navigation.njk" %}
+{% endblock %}
+
+{% block mainContent %}
+  <div class="govuk-grid-column-three-quarters">
+    <h1 class="govuk-heading-l">Team members</h1>
+    {% if isServiceAdmin %}
+      <p class="govuk-body govuk-!-margin-bottom-6 admin-lede">Team member changes affect both live and test environments.</p>
+    {% else %}
+      <p class="govuk-body govuk-!-margin-bottom-6 admin-lede">Contact an administrator to invite team members and change permissions.</p>
+    {% endif %}
+
+    {% if isServiceAdmin %}
+      {{
+      govukButton({
+        text: "Invite a team member",
+        href: inviteTeamMemberLink,
+        attributes: {
+          id: "invite-team-member-link"
+        }
+      })
+      }}
+    {% endif %}
+  </div>
+
+  <div class="govuk-grid-column-three-quarters admin-list-group govuk-!-margin-top-6">
+  {% for role, roleDetails in roles %}
+    {% if role in teamMembers %}
+      {% if teamMembers[role].length %}
```

Comment:
I think this could be simplified a bit. If you pass an array to view that's something like 
```
[
 {
   role: <admin role>,
   users: [<users>]
 },
 {
   role: <another role>,
   users: [<users>]
 },
  ... etc
]
```
you can order it correctly in the controller and only pass in roles for which there are users, then iterate that to produce the view without having to check each role for any users

Reply:
Yes that did occur to me. I'd prefer to leave further refactoring until later, but that may just reflect my exhaustion with it